src/ltfs_copyright.h

```diff
@@ -55,8 +55,8 @@ extern "C" {
 #endif
 
 #define LTFS_COPYRIGHT_0              "Licensed Materials - Property of IBM"
-#define LTFS_COPYRIGHT_1              "IBM Spectrum Archive Single Drive Edition"
-#define LTFS_COPYRIGHT_2              "(C) Copyright IBM Corp. 2011, 2017 All Rights Reserved"
+#define LTFS_COPYRIGHT_1              "IBM Storage Archive Single Drive Edition"
+#define LTFS_COPYRIGHT_2              "(C) Copyright IBM Corp. 2011, 2017, 2023 All Rights Reserved"
```

Comment:
Shall be 

`(C) Copyright IBM Corp. 2011, 2023`

First number shall be the year which is initially created, second number is the year which is last modified.

Reply:
Thank you, Abe-San
I've updated the copyright years as needed.